torch/csrc/distributed/rpc/init.cpp

```diff
@@ -473,7 +454,38 @@ PyObject* rpc_init(PyObject* /* unused */) {
 
   // Base class: torch.distributed.rpc.RpcBackendOptions.
   py::class_<TensorPipeRpcBackendOptions>(
-      module, "TensorPipeRpcBackendOptions", rpcBackendOptions)
+      module,
+      "TensorPipeRpcBackendOptions",
+      rpcBackendOptions,
+      R"(
+          The backend options for
+          :class:`~torch.distributed.rpc.TensorPipeAgent`, derived from
+          :class:`~torch.distributed.rpc.RpcBackendOptions`.
+
+          Arguments:
+              num_worker_threads (int, optional): The number of threads in the
+                  thread-pool used by
+                  :class:`~torch.distributed.rpc.TensorPipeAgent` to execute
+                  requests (default: 16).
+              _transports (list, optional): THIS IS AN ADVANCED PARAMETER, DO
+                  NOT USE UNLESS INSTRUCTED TO. The names of the TensorPipe
+                  transports that the agent should register, in order of
+                  decreasing priority.
+              _channels (list, optional): THIS IS AN ADVANCED PARAMETER, DO NOT
+                  USE UNLESS INSTRUCTED TO. The names of the TensorPipe channels
+                  that the agent should register, in order of decreasing
+                  priority.
```

Comment:
I was wondering: should we even include these, or should we hide them? It may make sense to hide them, because in the expected scenario (i.e., no transports/channels is defective) we don't want anyone to use them.

Reply:
+1 to hide them, if users really need it, we can then tell them there is such args here.